frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -160,9 +160,9 @@ action.GitMachete.CheckoutSelectedBranchAction.description.precise=Checkout bran
 action.GitMachete.CheckoutSelectedBranchAction.task-title=Checking out
 
 
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.text=Git Machete: Fast Forward Parent To Match Selected Branch
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContextMenu.text=_Fast Forward Parent To Match This Branch
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
+action.GitMachete.FastForwardParentToMatchSelectedBranchAction.text=Git Machete: Fast Forward Merge of Selected Branch To Parent
```

Comment:
I'd even skip this `of`, just `Fast-Forward Merge Selected Branch To Parent` (?)